src/system/behavior.cpp

```diff
@@ -243,16 +362,14 @@ void button_hold_callback(const uint8_t consecutiveButtonCheck,
                     BRIGHTNESS_RAMP_DURATION_MS * percentOfTimeToGoDown),
                 0, BRIGHTNESS_RAMP_DURATION_MS * percentOfTimeToGoDown,
                 currentBrightness, MIN_BRIGHTNESS);
+
         update_brightness(newBrightness);
```

Comment:
you can replace this line with `update_brightness(newBrightness, isEndOfHoldEvent);` and remove the test above